node-fs-writefile/note-to-self.js

```diff
@@ -0,0 +1,8 @@
+var fs = require('fs');
+const note = String(process.argv[2]);
+
+if (process.argv[2] !== undefined) {
+  fs.writeFile('note.txt', note, 'utf8', err => {
+    if (err) throw err;
+  });
+} else { throw Error; }
```

Comment:
When throwing a your own error, you should create a `new` error with a message about what went wrong.
```JAVASCRIPT
throw new Error('Missing note to write to file');
```